core/txdb/internal/storage/snapshot.proto

```diff
@@ -9,11 +9,11 @@ message Snapshot {
   // The nodes are ordered according to a pre-order traversal.
   repeated StateTreeNode nodes = 1;
 
-  // Issuances contains the record of recent issuances for ensuring uniqueness
-  // of issuances.
-  repeated Issuance issuances = 2;
```

Comment:
In this case I don't agree, but I did it anyway: https://github.com/chain/chain/pull/788/commits/af4e95f8df9c1324b23f97569ca5be9788da70ed